test/integration/accounting_spec.js

```diff
@@ -171,30 +171,42 @@ var mocker = function (database) {
         if (err) {
             console.log('Error initializing the database');
             process.exit(1);
+        } else {
+            return done();
         }
     });
 }
 
 console.log('[LOG]: starting an endpoint for testing...');
 test_endpoint.run(test_config.accounting_port);
 
-async.each(test_config.databases, function (database, task_callback) {
-
-    /**
-     * Remove the database used for testing.
-     */
-    after(function (task_callback) {
+after(function (done) {
+    async.eachSeries(test_config.databases, function (database, task_callback) {
         prepare_test.clearDatabase(database, databaseName, task_callback);
+    }, function (err) {
+        if (err) {
+            console.log(err);
+            return done();
+        } else {
+            return done();
+        }
```

Comment:
In the done is executed in both the if and the else you can extract it from the if